### ISSUE TITLE
Update app logo/icon

### ISSUE THREAD

**Issue by saalmaan** [yuliskov/SmartYouTubeTV#56]
Current app icon does not do justice to the this quality app. So, calling all designer to submit icons.

Legal question: can the app or its icon refernce YouTube ?

Thanks

Current icon/logo
![ca947e2e-220c-4a4f-bd92-1e81395d05a6](https://user-images.githubusercontent.com/4620730/34905616-bad3743a-f854-11e7-9607-b6918bb74aeb.jpeg)

**Comment by yuliskov** [comment]
@saalmaan yup, icon must be distinct from the original YouTube one. This requirement is mandatory.

**Comment by SECURIOlab** [comment]
much better wil be Y instead of S letter inside tv icon

**Comment by saalmaan** [comment]
@TravisHall it will be great if you can design us a decent logo

**Comment by yuliskov** [comment]
@saalmaan thanks. Sorry for late reply - have much work.  
I think that icon must be unique and somehow reflect the purpose of the application. This icon reminds me about Android Lollipop release btw.

**Comment by saalmaan** [comment]
Welcome back @yuliskov I agree this somewhat resembles Android Lollipop. I will get my designer to help us with logo

**Comment by yuliskov** [comment]
Does anyone have propositions with logo?
